0.data-download/scripts/data_loader.py

```diff
@@ -12,69 +12,78 @@ def load_data(data_directory, adult_or_pediatric="all"):
     )
     dependency_data_file = pathlib.Path(data_directory, "CRISPRGeneDependency.csv")
 
     # Load data
     dependency_df = (
-        pd.read_csv(dependency_data_file, index_col=0).reset_index().dropna(1)
+        pd.read_csv(dependency_data_file, index_col=0).reset_index().dropna(axis=1)
     )
-    sample_df = pd.read_csv(sample_info_file, index_col=0)
+    model_df = pd.read_csv(model_file, index_col=0)
 
-    # rearrange sample info and gene dependency dataframe indices so DepMap_IDs are in alphabetical order
-    sample_df = sample_df.sort_index(ascending=True)
-    sample_df = sample_df.reset_index()
+    # rearrange model info and gene dependency dataframe indices so DepMap_IDs are in alphabetical order
+    model_df = model_df.sort_index(ascending=True)
+    model_df = model_df.reset_index()
 
     dependency_df = (
         dependency_df.set_index("DepMap_ID").sort_index(ascending=True).reset_index()
     )
 
-    # searching for similar IDs FROM dependency df IN sample df
+    # searching for similar IDs FROM dependency df IN model df
     dep_ids = dependency_df["DepMap_ID"].tolist()
-    dep_vs_samp_ids = set(dep_ids) & set(sample_df["DepMap_ID"].tolist())
+    dep_vs_mod_ids = set(dep_ids) & set(model_df["DepMap_ID"].tolist())
 
-    # searching for similar IDs FROM sample df In dependency df
-    samp_ids = sample_df["DepMap_ID"].tolist()
-    samp_vs_dep_ids = set(samp_ids) & set(sample_df["DepMap_ID"].tolist())
+    # searching for similar IDs FROM model df In dependency df
+    mod_ids = model_df["DepMap_ID"].tolist()
+    mod_vs_dep_ids = set(mod_ids) & set(model_df["DepMap_ID"].tolist())
 
-    # subset data to only matching IDs (samples in both dependency and sample data)
-    sample_df = sample_df.loc[sample_df["DepMap_ID"].isin(dep_vs_samp_ids)].reset_index(
+    # subset data to only matching IDs (samples in both dependency and model data)
+    model_df = model_df.loc[model_df["DepMap_ID"].isin(dep_vs_mod_ids)].reset_index(
         drop=True
     )
-    dependency_df = dependency_df.loc[dependency_df["DepMap_ID"].isin(samp_vs_dep_ids)]
+    dependency_df = dependency_df.loc[dependency_df["DepMap_ID"].isin(mod_vs_dep_ids)]
 
     if adult_or_pediatric != "all":
-        sample_df = sample_df.query(
+        model_df = model_df.query(
             "age_categories == @adult_or_pediatric"
         ).reset_index(drop=True)
-        samples_to_keep = sample_df.reset_index(drop=True).DepMap_ID.tolist()
+        model_to_keep = model_df.reset_index(drop=True).DepMap_ID.tolist()
         dependency_df = dependency_df.query(
             "DepMap_ID == @samples_to_keep"
         ).reset_index(drop=True)
 
-    return sample_df, dependency_df
+    return model_df, dependency_df
+    
 
-
-def load_train_test_data(data_directory, train_or_test="all"):
+def load_train_test_data(data_directory, trainf="VAE_train_df.csv", testf="VAE_test_df.csv", train_or_test="all", stats=False):
```

Comment:
```suggestion
def load_train_test_data(data_directory, train_file="VAE_train_df.csv", test_file="VAE_test_df.csv", train_or_test="all", load_gene_stats=False):
```

Just to be super clear about what the variables contain :)

Reply:
you'll need to change the variable name throughout this script